sw/device/lib/dif/dif_uart.c

```diff
@@ -98,6 +113,9 @@ static bool uart_configure(const dif_uart_t *uart,
     return false;
   }
 
+  // Must be called before the first write to any of the UART registers
```

Comment:
I'd actually prefer that type of comment next to the function definition, not next to the call site.